code/control/DNDeploymentHandlers.php

```diff
@@ -5,15 +5,20 @@
 class DNDeploymentHandlers extends Object {
 
 	public function onSubmit(TransitionEvent $e) {
+		/** @var DNDeployment $deployment */
 		$deployment = $e->getStateMachine()->getObject();
+		$deployment->DeployRequested = SS_Datetime::now()->Rfc2822();
```

Comment:
"Deploy date" in the history - is it the start, or the _end_ of the deployment job? :)